numpy/lib/tests/test_io.py

```diff
@@ -575,13 +576,9 @@ def test_unicode_and_bytes_fmt(self, fmt, iotype):
     @pytest.mark.skipif(sys.platform=='win32',
                         reason="large files cause problems")
     @pytest.mark.slow
+    @requires_memory(7e9)
```

Comment:
nit - the original spelling seemed less magic, and had the bonus of being integral
```suggestion
    @requires_memory(6 * 1024 * 1024 * 1024)
```